### PR TITLE
Added release notes for 2020.13

### LINKED ISSUE #4956: Release notes 2020.13
**Describe the issue. What is the expected and unexpected behavior?**
release notes 2020.13
**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**

**What is your product and what release version are you targeting?**

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #4956

**Additional issues**:

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+- [@patternfly/react-catalog-view-extension@4.8.59](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.8.59)
+- [@patternfly/react-charts@6.10.4](https://www.npmjs.com/package/@patternfly/react-charts/v/6.10.4)
+- [@patternfly/react-core@4.63.2](https://www.npmjs.com/package/@patternfly/react-core/v/4.63.2)
+- [@patternfly/react-icons@4.7.11](https://www.npmjs.com/package/@patternfly/react-icons/v/4.7.11)
+- [@patternfly/react-inline-edit-extension@4.5.116](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.5.116)
+- [@patternfly/react-styles@4.7.8](https://www.npmjs.com/package/@patternfly/react-styles/v/4.7.8)
+- [@patternfly/react-table@4.18.13](https://www.npmjs.com/package/@patternfly/react-table/v/4.18.13)
+- [@patternfly/react-tokens@4.9.12](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.9.12)
+- [@patternfly/react-topology@4.6.24](https://www.npmjs.com/package/@patternfly/react-topology/v/4.6.24)
+- [@patternfly/react-virtualized-extension@4.5.104](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.104)
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:17]: Versions are a little off. Should be:
```json
"@patternfly/react-catalog-view-extension": "4.8.60",
"@patternfly/react-charts": "6.10.4",
"@patternfly/react-core": "4.63.3",
"@patternfly/react-icons": "4.7.11",
"@patternfly/react-inline-edit-extension": "4.5.117",
"@patternfly/react-styles": "4.7.8",
"@patternfly/react-table": "4.18.14",
"@patternfly/react-tokens": "4.9.12",
"@patternfly/react-topology": "4.6.25",
"@patternfly/react-virtualized-extension": "4.5.105",
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+  - Passed aria-label / aria-labelledby to the `<ul role="listbox">` to prevent axe issues ([#4931](https://github.com/patternfly/patternfly-react/pull/4931))
+  - Updated to remove the clear button button when onClear function is not specified for typeahead ([#4887](https://github.com/patternfly/patternfly-react/pull/4887))
+- **Tabs:** 
+  - Added support for light scheme variant in Tabs && TabContent ([#4832](https://github.com/patternfly/patternfly-react/pull/4832))
+  - Conditionally render aria-controls when mountOnEnter ([#4796](https://github.com/patternfly/patternfly-react/pull/4796))
+- **Tooltip** 
+  - Updated to export TooltipProps as a type to help babel ([#4854](https://github.com/patternfly/patternfly-react/pull/4854))
+
+### Catalog view extension:** 
+  - Added back catalogue items ([#4940](https://github.com/patternfly/patternfly-react/pull/4940))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:70]: Shouldn't be indented.

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+- Replaced or used exact lodash imports ([#4830](https://github.com/patternfly/patternfly-react/pull/4830))
+
+### Topology
+- Fixed exports ([#4927](https://github.com/patternfly/patternfly-react/pull/4927))
+- Fixed selector for cancelling dnd ([#4850](https://github.com/patternfly/patternfly-react/pull/4850))
+- Updated to wrap drag event handlers in actions ([#4826](https://github.com/patternfly/patternfly-react/pull/4826))
+
+### PF3
+- **Vertical nav:**
+  - Updates to not handle modified clicks on VerticalNav links ([#4747](https://github.com/patternfly/patternfly-react/pull/4747))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:90]: Can we put PF3 below Other? It's a little confusing.

**Review by redallen — CHANGES_REQUESTED** [review]

File: packages/react-docs/RELEASE-NOTES.md (hunk 3 above)
Comment (tlabaj) [reply packages/react-docs/RELEASE-NOTES.md:90]: yes

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+  - Updated to return the string value for select chips ([#4915](https://github.com/patternfly/patternfly-react/pull/4915))
+  - Passed aria-label / aria-labelledby to the `<ul role="listbox">` to prevent axe issues ([#4931](https://github.com/patternfly/patternfly-react/pull/4931))
+  - Updated to remove the clear button button when onClear function is not specified for typeahead ([#4887](https://github.com/patternfly/patternfly-react/pull/4887))
+- **Tabs:** 
+  - Added support for light scheme variant in Tabs && TabContent ([#4832](https://github.com/patternfly/patternfly-react/pull/4832))
+  - Conditionally render aria-controls when mountOnEnter ([#4796](https://github.com/patternfly/patternfly-react/pull/4796))
+- **Tooltip** 
+  - Updated to export TooltipProps as a type to help babel ([#4854](https://github.com/patternfly/patternfly-react/pull/4854))
+
+### Catalog view extension:**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:69]: Remove asterisks

```suggestion
### Catalog view extension:
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+  - Added support for inline link variation that acts like a link ([#4859](https://github.com/patternfly/patternfly-react/pull/4859))
+- **Chip group:**
+  - Added main div to chip group to fix close button wrapping ([#4873](https://github.com/patternfly/patternfly-react/pull/4873))
+- **Data list:** 
+  - Added draggable feature for basic list ([#4852](https://github.com/patternfly/patternfly-react/pull/4852))
+  - Added support for text modifiers ([#4846](https://github.com/patternfly/patternfly-react/pull/4846))
+  - Added ref to omit list ([#4948](https://github.com/patternfly/patternfly-react/pull/4948))
+  - Updated aria of data list ([#4913](https://github.com/patternfly/patternfly-react/pull/4913))
+- **Date picker:** 
+  Added basic date picker ([#4888](https://github.com/patternfly/patternfly-react/pull/4888))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:32]: Add bullet point to break to new line

```suggestion
  - Added basic date picker ([#4888](https://github.com/patternfly/patternfly-react/pull/4888))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 6]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+  - Added OUIA support into nav item ([#4769](https://github.com/patternfly/patternfly-react/pull/4769))
+- **Notification drawer:** 
+  - Added default variation to notification items ([#4792](https://github.com/patternfly/patternfly-react/pull/4792))
+  - Removed unused import ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Page:** 
+  - Replaced hasNoPadding property in the documentation example ([#4848](https://github.com/patternfly/patternfly-react/pull/4848))
+  - Fixed handleMainClick function signature ([#4843](https://github.com/patternfly/patternfly-react/pull/4843))
+- **Progress:** 
+  - Added warning variant ([#4834](https://github.com/patternfly/patternfly-react/pull/4834))
+- **Progress:**
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:53]: Can remove duplicate "Progress" heading and move this bullet up with the other Progress item

```suggestion

```

File: packages/react-docs/RELEASE-NOTES.md (hunk 5 above)
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:32]: Needs a bullet point

**Review by redallen — CHANGES_REQUESTED** [review]
Caught one more little thing

File: packages/react-docs/RELEASE-NOTES.md [hunk 7]
```diff
@@ -4,6 +4,114 @@ section: get-started
...
+  - Updated to export TooltipProps as a type to help babel ([#4854](https://github.com/patternfly/patternfly-react/pull/4854))
+
+### Catalog view extension:** 
+- Added back catalogue items ([#4940](https://github.com/patternfly/patternfly-react/pull/4940))
+
+### Charts
+- Replaced or used exact lodash imports ([#4830](https://github.com/patternfly/patternfly-react/pull/4830))
+
+### Icons
+- Used lying cast ([#4935](https://github.com/patternfly/patternfly-react/pull/4935))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:76]: Do we want a more descriptive summary here, such as "Revert support for svg alt tags"?

**Review by evwilkin — APPROVED** [review]
A few small tweaks & fixes suggested

**Review by evwilkin — CHANGES_REQUESTED** [review]
A few small tweaks & fixes suggested

**Review by evwilkin — APPROVED** [review]
LGTM - thanks!

**Review by redallen — APPROVED** [review]

**Pull request merged**